aws/resource_aws_lakeformation_resource.go

```diff
@@ -0,0 +1,105 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lakeformation"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceAwsLakeFormationResource() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsLakeFormationResourceRegister,
+		Read:   resourceAwsLakeFormationResourceDescribe,
+		Delete: resourceAwsLakeFormationResourceDeregister,
+
+		Schema: map[string]*schema.Schema{
+			"resource_arn": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
```

Comment:
Maybe `validateArn` is better?

Reply:
Indeed, thanks for mentioning this function.